### DISCUSSION TITLE
IBM Semeru Runtime binary eating up CPU when VS Code is running

### DISCUSSION THREAD

**Discussion by gcraig88keys** [codefori/vscode-ibmi#2271] [General]
![IBM Semeru](https://github.com/user-attachments/assets/1fd5adef-390b-4258-aa36-f2a7b0425fed)

IBM Semeru Java runtime binary is hogging CPU on Windows 10 when VS Code IBM i plugins are running.   Any ideas on how to resolve this.   It's causing all kinds of delays.   I can barely type.   Sometimes my PC will hang for up to 30 or 45 seconds.  This just started recently in the last couple of weeks.   Could something have changes with one of the plugins?

**Comment by sebjulliand** [comment]
Code for i doesn't include IBM's Semeru JVM or trigger any Java process; I doubt it's related to the extension.
Could you check the process details to see where Semeru is installed? And maybe see the command line what it's running...it may help pinpoint the issue.
